content/1.2-rc/docs/installation/requirements.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Requirements"
+date: 2024-11-25
+---
+
+The orchestrator runtime/deployment is made of two main parts: `sonataflow operator` and `RHDH operator`  
+SonataFlow operator resource requirements are described [here](https://sonataflow.org/serverlessworkflow/latest/cloud/operator/install-serverless-operator.html#_prepare_a_minikube_instance)  
```

Comment:
Instead of `[here]` I would suggest to use a label similar to `SonataFlow Installation Requirements` to clarify that's an external requirement

Reply:
fixed